docs/en/CSMM/advanced-feature-guide-deadlydrops.md

```diff
@@ -21,32 +21,14 @@ zcoord :    (?<=Spawned supply crate at \(-?\d{1,4}\.\d, -?\d{1,4}\.\d, )(-?\d{1
 
 Note that we are avoiding the decimal points in the coordinates and only grabbing the integers, but when including the coordinates in the lookbehind for the zcoord variable we do need to account for the extra characters.
 
-We will use these coordinates to identify the entityId of the supply crate, which will use to create a unique name for the advanced claim in the next step. To get this, our command for this hook will be `w2l command "lce ${custom.xcoord} ${custom.zcoord} 5 SupplyCrate"`. This will look up any entities of type SupplyCrate within 5 blocks of the location of the airdrop, and will output a line including the coordinates again plus the supply crate ID.
+We will use these coordinates to create the claim, but we also need to make sure that the next airdrop doesn't try to create a claim with the same name. For this, we will use the `{{time}}` variable. This outputs the real-world time when the command is run, and it is unlikely that another supply drop will happen at exactly the same time on another day. We'll call our claim `DeadlyDrop{{time}}`.
 
-## The hook to create the advanced claim
+## The 'command' advanced claim
 
-Now we need another hook to listen for the output of the `w2l` from our first hook. The log line for this will be
+Now we have the coordinates and a unique name, we need to put these together to create the command claim. This will trigger our second hook when it someone enters the area. We'll use the coordinates of the crate plus and minus 20 to create a square 40 blocks by 40 blocks around it. The command the claim will execute is simply to log a keyword along with the victim's steamID for our next hook, and then delete itself. The command for this is as follows:
```

Comment:
```suggestion
Now we have the coordinates and a unique name, we need to put these together to create the command claim. This will trigger our second hook when someone enters the area. We'll use the coordinates of the crate plus and minus 20 to create a square 40 blocks by 40 blocks around it. The command the claim will execute is simply to log a keyword along with the victim's steamID for our next hook, and then delete itself. The command for this is as follows:
```